USAGEGUIDE.md

```diff
@@ -5,114 +5,61 @@
 
-## Deploying 1Password Connect to Kubernetes
-
-If 1Password Connect is already running, you can skip this step.
-
-There are options to deploy 1Password Connect:
-
-- [Deploy with Helm](#deploy-with-helm)
-- [Deploy using the Connect Operator](#deploy-using-the-connect-operator)
-
-### Deploy with Helm
-
-The 1Password Connect Helm Chart helps to simplify the deployment of 1Password Connect and the 1Password Connect Kubernetes Operator to Kubernetes.
-
-[The 1Password Connect Helm Chart can be found here.](https://github.com/1Password/connect-helm-charts)
-
-### Deploy using the Connect Operator
-
-This guide will provide a quickstart option for deploying a default configuration of 1Password Connect via starting the deploying the 1Password Connect Operator, however, it is recommended that you instead deploy your own manifest file if customization of the 1Password Connect deployment is desired.
-
-Encode the `1password-credentials.json` file you generated in the prerequisite steps and save it to a file named `op-session`:
-
-```bash
-cat 1password-credentials.json | base64 | \
-  tr '/+' '_-' | tr -d '=' | tr -d '\n' > op-session
-```
-
-Create a Kubernetes secret from the op-session file:
-
-```bash
-kubectl create secret generic op-credentials --from-file=op-session
-```
-
-Add the following environment variable to the onepassword-connect-operator container in `/config/manager/manager.yaml`:
-
-```yaml
-- name: MANAGE_CONNECT
-  value: "true"
-```
-
-Adding this environment variable will have the operator automatically deploy a default configuration of 1Password Connect to the current namespace.
-
-## Kubernetes Operator Deployment with Connect
-
-#### Create Kubernetes Secret for OP_CONNECT_TOKEN ####
-
-Create a Connect token for the operator and save it as a Kubernetes Secret:
+---
 
-```bash
-kubectl create secret generic onepassword-token --from-literal=token="<OP_CONNECT_TOKEN>"
-```
+##  Use Kubernetes Operator with Service Account
 
-If you do not have a token for the operator, you can generate a token and save it to Kubernetes with the following command:
+### 1. [Create a service account](https://developer.1password.com/docs/service-accounts/get-started#create-a-service-account)
+### 2. Create a Kubernetes secret for the Service Account
+- Set `OP_SERVICE_ACCOUNT_TOKEN` environment variable to the service account token you created in the previous step. This token will be used by the operator to access 1Password items.
+- Create Kubernetes secret:
 
 ```bash
-kubectl create secret generic onepassword-token --from-literal=token=$(op create connect token <server> op-k8s-operator --vault <vault>)
+kubectl create secret generic onepassword-service-account-token --from-literal=token="$OP_SERVICE_ACCOUNT_TOKEN"
 ```
 
-**Deploying the Operator**
+### 3. Deploy the Operator
```

Comment:
The instructions here assume that you deploy it using the repository. In practice, the deployment is the same as doing it with Connect. The only difference is the operator's config which now points to a service account rather than Connect.

Reply:
That's true, there are some similarly between them. As result you run `make deploy` command for both.

Reply:
What I'm trying to get at is consistency. Therefore, we should point to deployment of the operator with Helm or otherwise just like we do with Connect. Having a completely different method (local repo) can throw some people off.

Reply:
We'll update it afterwards, with the link to the dev portal documentation, right after the release.